plugins/plugin-babel/test/plugin-babel.test.js

```diff
@@ -0,0 +1,117 @@
+const plugin = require('../plugin.js');
+const babel = require('@babel/core');
+const workerpool = require('workerpool');
+
+jest.mock('@babel/core');
+babel.transformFileAsync = jest.fn(() => Promise.resolve({code: 'code', map: 'map'}));
+
+/** jest mock above in worker will not work */
+jest.mock('workerpool');
```

Comment:
Same here—do you get better results mocking this before importing `workerpool` (or better yet—can you mock it without importing it?)